funnel/templates/macros.html.jinja2

```diff
@@ -691,7 +690,7 @@
                   <li>
                     {{ proposal.state.label.title }}
                   </li>
-                  <li class="no-border">{{ proposal.datetime|datetime(format='EEE, dd MMM') }}</li>
+                  <li class="no-border">{{ proposal.datetime|datetime(format='EEE, dd MMM YYYY') }}</li>
```

Comment:
Also see the Up Next card and the proposal main body. Date format switches between `dd MMM` and `MMM dd`.